about.html

```diff
@@ -0,0 +1,349 @@
+                  <button
+                    type="button"
+                    class="nav-bar-icon-logo-li transition font"
+                  >
+                    Logout
+                  </button>
+                </li>
+              </ul>
+              <div class="nav-bar-logo-div">
+                <section class="logo">
+                  <a class="header-logo transition font" href="index.html"
+                    >©️ComedyBurst</a
+                  >
+                </section>
+                <ul>
+                  <li>
+                    <a
+                      class="bottom-header-ul-li-text transition font"
+                      href="index.html"
+                      >Home</a
+                    >
+                  </li>
+                  <li>
+                    <a class="bottom-header-ul-li-text transition font" href="#"
+                      >Event</a
+                    >
+                  </li>
+                  <li>
+                    <a class="bottom-header-ul-li-text transition font" href="#"
+                      >Join</a
+                    >
+                  </li>
+                  <li>
+                    <a class="bottom-header-ul-li-text transition font" href="#"
+                      >Sponsor</a
+                    >
+                  </li>
+                  <li>
+                    <a class="bottom-header-ul-li-text transition font" href="#"
+                      >News</a
+                    >
+                  </li>
+                </ul>
+                <button type="button" class="campaign transition font">
+                  ComedyBurst Initiative
+                </button>
+              </div>
+            </section>
+          </div>
+        </div>
+        <div class="about-page-header-body for-phone">
+          <section class="about-header-text-div">
+            <h1 class="about-header-text-head font">"Hey! Comic lovers"</h2>
+            <h2 class="about-header-text-body font">
+              FIND THE BEST COMIC WHEREVER YOU ARE
+            </h2>
+          </section>
+          <section class="about-header-text-description-container">
+            <p class="about-header-text-description font">
+              Discover the best comics, matches to your comdey taste.
+            </p>
+          </section>
+          <section class="contact-paragraph-div">
+            <h5 class="contact-paragraph font">
+              Please contact us per Email for any further questions about
+              ComedyBurst
+            </h5>
+          </section>
+          <section class="email-link-div">
+            <a class="email-link" href="#">ComedyBurst@comic.org</a>
+          </section>
+        </div>
+        <div class="about-page-header-body-desk for-desktop">
+          <section class="about-header-text-div-desk">
+            <h1 class="about-header-text-body-desk font">
+              FIND THE BEST COMIC WHEREVER YOU ARE
+            </h1>
```

Comment:
- [ ] Kindly note that the `h1` tag should be used only once on each page, kindly use another `hx`.

Reply:
Done 👍 @Ofarouq310